examples/forms.amp.html

```diff
@@ -270,12 +270,16 @@
       </template>
     </div>
     <div submit-success>
-      Success! Thanks for subscribing! Please make sure to check your email
```

Comment:
Interesting! Glad we both came across this because the validator still requires there to be a template tag. Filed as #9892